src/content/app/genome-browser/hooks/useFocusTrack.ts

```diff
@@ -103,8 +93,8 @@ const useFocusGene = (params: Params) => {
 
   useEffect(() => {
     focusObjectIdRef.current = focusObjectId;
-    geneIdRef.current = parsedFocusObjectId?.objectId;
-  }, [focusObjectId, parsedFocusObjectId?.objectId]);
+    geneIdRef.current = focusGene?.stable_id;
```

Comment:
Why not use `geneStableId` here, defined above, instead?

Reply:
Good point. Updated.